tests/probes/fwupdsecattr/test_probes_fwupdsecattr_mock.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+. $builddir/tests/test_common.sh
+
+set -e -o pipefail
+
+function test_probes_fwupdsecattr {
+
+    probecheck "fwupdsecattr" || return 255
+
+    local ret_val=0
+    local DF="${srcdir}/test_probes_fwupdsecattr_mock.xml"
+    local RF="results.xml"
+    local DBUS_MOCK_NAME="org.freedesktop.fwupd"
+
+    [ -f $RF ] && rm -f $RF
+
+    init_dbus_mock $DBUS_MOCK_NAME
+    $OSCAP oval eval --results $RF $DF
+    clean_dbus_mock $DBUS_MOCK_NAME
+
+    if [ -f $RF ]; then
+        verify_results "def" $DF $RF 1 && verify_results "tst" $DF $RF 1
+        ret_val=$?
+    else
+        ret_val=1
+    fi
+
```

Comment:
If the warnings like `W: oscap:     HSI key not found: org.fwupd.hsi.Kernel.InvalidOrNonExisting` are expected here, there should be an assert that tests them.

Reply:
Done.